### DISCUSSION TITLE
Como entregar la Kata de Abogabot

### DISCUSSION THREAD

**Discussion by Eholguin05** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#59] [Q&A] [answered]
Hola que tal, como se entregara la practica de abogabot?
Gracias por su tiempo!

**Comment by RaulAcevedoF** [comment, marked as answer]
Que tal @Eholguin05 para la práctica de abogabot si estas trabajando con la plantilla que propuso el MC tendrías que subir ese documento o si estas trabajando con un diseño propio seria conveniente que subieras ese documento a tu repositorio (ya sea uno nuevo o como lo estes trabajando).    
En el último live el MC comento que nuevamente se manejaría un formulario para poder compartir el link de nuestro repositorio, me imagino que más adelante lo comentará nuevamente.
Espero haberte ayudado.

**Reply by Eholguin05** [reply]
Muchas gracias!

**Reply by JuanSalvador73** [reply]
Según entiendo, bajamos el formulario, lo llenamos y lo subimos. Aunque en el mismo hay puntos que no están muy claros. al menos para quienes no lo hemos hecho así previamente.

**Reply by JLuisPrz** [reply]
No es necesario llenar todos los puntos mencionados en el documento, además puedes hacer tu propio diseño de levantamiento de requerimientos.

**Reply by krlosbyte** [reply]
Yo he buscado plantillas, pero parece que no hay algo simple y claro, todos son largos y tediosos.
